frontend/sozisel-app/src/components/SessionDetails/SessionEditing/SessionEditing.tsx

```diff
@@ -1,36 +1,39 @@
-import "./EditSession.scss";
+import "./SessionEditing.scss";
 
 import MuiAlert, { AlertProps } from "@material-ui/lab/Alert";
 import { ReactElement, useState } from "react";
+import SessionCustomization, {
+  onSubmitProps,
+} from "../SessionCustomization/SessionCustomization";
 import {
   useSessionDetailsQuery,
   useUpdateSessionMutation,
 } from "../../../graphql";
 
+import { AUTO_HIDE_DURATION } from "../../../common/consts";
 import CircularProgress from "@material-ui/core/CircularProgress";
 import MainNavbar from "../../MainNavbar/MainNavbar";
-import SessionCustomization from "../SessionCustomization/SessionCustomization";
 import Snackbar from "@material-ui/core/Snackbar";
 import TemplateOverview from "../TemplateOverview/TemplateOverview";
-import { useLocation } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 
 function Alert(props: AlertProps) {
   return <MuiAlert elevation={6} variant="filled" {...props} />;
 }
 
-export interface SessionEditProps {
+export interface SessionEditingProps {
   sessionId: string;
 }
 
-export default function EditSession(): ReactElement {
+export default function SessionEditing(): ReactElement {
```

Comment:
EditSession and CreateSession looks much cleaner. And I don't know what to think about `SessionCustomization` name.